pkg/env/env.go

```diff
@@ -15,13 +15,12 @@
 package env
 
 import (
+	"encoding/base64"
 	"fmt"
-	"html/template"
 	"os"
 	"path/filepath"
 	"strings"
-
-	"github.com/Masterminds/sprig/v3"
```

Comment:
You removed `github.com/Masterminds/sprig/v3`?  Just remind that if you need to `go mode tidy` to remove that from go.mod? @VinceCui

Reply:
it used some where.